FindSymbolOccurance.java

```diff
@@ -1,16 +1,17 @@
 package hw5;
 
-public class findSymbolOccurance {
+public class FindSymbolOccurance {
     public static int countOccurrences(String str, char c) {
```

Comment:
What will happen if 'str' == null? Could you please add null validation for the 'str' parameter?